src/braket/pennylane_plugin/translation.py

```diff
@@ -82,11 +82,11 @@ def supported_operations(device: Device) -> FrozenSet[str]:
         properties = device.properties.action["braket.ir.jaqcd.program"]
     except AttributeError:
         raise AttributeError("Device needs to have properties defined.")
-    supported_ops = frozenset(op.lower() for op in properties.supportedOperations)
+    supported_ops = frozenset(op.lower().replace("_", "") for op in properties.supportedOperations)
     return frozenset(
         _BRAKET_TO_PENNYLANE_OPERATIONS[op]
         for op in _BRAKET_TO_PENNYLANE_OPERATIONS
-        if op.lower() in supported_ops
+        if op.lower().replace("_", "") in supported_ops
     )
```

Comment:
Why do we still need to perform `.lower().replace()` ?

Reply:
Just paranoid future-proofing

Reply:
There is a potential for issues due to this. For instance, `a_bc` and `ab_c` are treated the same even though they aren't. So if we have two operations which share the same letter sequence, but only differ by how those sequences are delineated, this would not work correctly.

Reply:
Thing is, the snake_case names `a_bc` and `ab_c` will almost definitely correspond to camelCase names `ABc` and `AbC` respectively, which `.lower()` will also fail to disambiguate. Given that all of our classes seem to have both a snake_case and CamelCase form, it's exceedingly unlikely that this conflict will occur.

Granted, the `replace` is unnecessary here, so I can safely remove it.